code/controllers/subsystem/garbage.dm

```diff
@@ -363,71 +363,77 @@ SUBSYSTEM_DEF(garbage)
 /// Should be treated as a replacement for the 'del' keyword.
 ///
 /// Datums passed to this will be given a chance to clean up references to allow the GC to collect them.
-/proc/qdel(datum/D, force=FALSE, ...)
+/proc/qdel(datum/D, force = FALSE, ...)
```

Comment:
I WAS JUST HEDGING MY BETS PUT IT BACK NOOO THE SOULCODE IT BURNS

Reply:
FINE